lib/ansible/modules/cloud/google/gcp_appengine_firewall_rule_info.py

```diff
@@ -44,8 +44,8 @@
 '''
 
 EXAMPLES = '''
-- name: " a firewall rule facts"
-  gcp_appengine_firewall_rule_facts:
+- name: " a firewall rule info"
```

Comment:
These auto-generated `name` lines are a bit strange - could you change them to use the pattern "get info on a/an {{ item }}"?